gpflow/posteriors.py

```diff
@@ -549,44 +477,45 @@ def _conditional_with_precompute(
         """
         assert_params_false(self._conditional_with_precompute, full_output_cov=full_output_cov)
 
-        alpha, Qinv = cache
+        L, LB, c = cache
 
         Kus = Kuf(self.inducing_variable, self.kernel, Xnew)
-        Knn = self.kernel(Xnew, full_cov=full_cov)
-
-        Ksu = tf.transpose(Kus)
-        mean = Ksu @ alpha
-
+        tmp1 = tf.linalg.triangular_solve(L, Kus, lower=True)
+        tmp2 = tf.linalg.triangular_solve(LB, tmp1, lower=True)
```

Comment:
Would it be possible to give these more meaningful names?

Reply:
> Would it be possible to give these more meaningful names?

I have no clue what to call them, though - and this is just copy n' pasted code, it might be better to keep it like this to make it easier to see that it's copy n' pasted.